src/main/scala/za/co/absa/hyperdrive/trigger/persistance/SensorRepository.scala

```diff
@@ -16,13 +16,15 @@
 package za.co.absa.hyperdrive.trigger.persistance
 
 import org.springframework.stereotype
-import za.co.absa.hyperdrive.trigger.models.Sensor
+import za.co.absa.hyperdrive.trigger.models.{Properties, Sensor}
```

Comment:
You can remove: 
import za.co.absa.hyperdrive.trigger.models.Properties
import za.co.absa.hyperdrive.trigger.models.enums.SensorTypes.SensorType